doc/source/whatsnew/v0.23.0.txt

```diff
@@ -176,6 +176,37 @@ Please note that the string `index` is not supported with the round trip format,
    new_df
    print(new_df.index.name)
 
+.. _whatsnew_0230.enhancements.index_division_by_zero
+
+Index Division By Zero Fills Correctly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Division operations on ``Index`` and subclasses will now fill positive / 0 with ``np.inf``, negative / 0 with ``-np.inf``, and 0 / 0 with ``np.nan``.  This matches existing Series behavior.
+
+Current Behavior:
+
+.. code-block:: ipython
+
+    In [3]: index = pd.Index([-1, 0, 1])
+    In [4]: index / 0
+    Out[4]: Int64Index([0, 0, 0], dtype='int64')
+    In [5]: index / 0.0
+    Out[5]: Float64Index([-inf, nan, inf], dtype='float64')
+
+    In [6]: index = pd.UInt64Index([0, 1])
+    In [7]: index / np.array([0, 0], dtype=np.uint64)
+    Out[7]: UInt64Index([0, 0], dtype='uint64')
+
+Previous Behavior:
+
+.. code-block:: ipython
```

Comment:
code blocks here are fine, but you need to actually show the executed code as in an ipython session.

Reply:
Not totally clear on the difference.  I'll be pushing shortly with fixes to most other comments and a best-guess as to these.